src/shared/views/navigation/NavigationItem/NavigationItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+import styles from './navigationItem.css'
+
+const NavigationItem = (props) => {
+  return (
+    <li>
+      <Link to={props.link}>
+        <div className={styles.navigationItem}>
+          <i className={`fa ${props.icon} ${styles.icon}`} aria-hidden="true" />
```

Comment:
Let's not hard code the fa into the class. Can we make the icon in navlink pass down 'fa fa-home' ?